### PR TITLE
Router

### PR DESCRIPTION
**Pull request by pariyar07**
Router and Navlink added

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1 +1,68 @@
...
+    font-family: 'Open Sans', sans-serif;
+}
+
+li{
+    list-style: none;
+}
+
+a{
+    text-decoration: none;
+}
```
Comment (sharmayatendra) [inline src/index.css:32]: try to avoid generic styling to element give it a class then style it

**Review by sharmayatendra — APPROVED** [review]
there are many styles.css files that are empty if they are not required plz remove that.
left few suggestions can look upon that
Good Work

**Review by tushargoyal812 — COMMENTED** [review]
folder structure is good.
variable naming is good.
looks good to merge.

File: src/index.css (hunk 1 above)
Comment (pariyar07) [reply src/index.css:32]: Thanks for the suggestions.

**Pull request merged**